pkg/oc/cli/cli.go

```diff
@@ -223,6 +225,7 @@ func NewOcCommand(name, fullName string, in io.Reader, out, errout io.Writer) *c
 				kubectlwrappers.NewCmdApiVersions(fullName, f, ioStreams),
 				kubectlwrappers.NewCmdApiResources(fullName, f, ioStreams),
 				kubectlwrappers.NewCmdClusterInfo(fullName, f, ioStreams),
+				diff.NewCmdDiff(f, ioStreams),
```

Comment:
Nit, we'll need a similar wrapper, or I'll just fix those bits in upstream, followup.

Reply:
> Nit, we'll need a similar wrapper, or I'll just fix those bits in upstream, followup.

Yeah, seems like you want a better plan overall than wrapping every single one.

Reply:
Yeah, I hate that, and with every new command it's just getting worse.